### PR TITLE
Comment-Book

### PR DESCRIPTION
**Pull request by MinhDo2007**

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
+  def create
+    @review = Review.find(params[:review_id])
```
Comment (MichalPlachyFramgia) [inline app/controllers/comments_controller.rb]: remove ()

File: app/views/comments/_comment.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div>
+<hr>
```
Comment (MichalPlachyFramgia) [inline app/views/comments/_comment.html.erb]: indent

File: app/views/comments/_comment.html.erb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+<span><%= link_to comment.user.email, comment.user %></span>
```
Comment (MichalPlachyFramgia) [inline app/views/comments/_comment.html.erb]: indent

File: app/views/comments/_comment.html.erb [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+<span class="content">
```
Comment (MichalPlachyFramgia) [inline app/views/comments/_comment.html.erb]: indent

File: app/views/comments/_comment.html.erb [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+  <%= comment.content %><br>
```
Comment (MichalPlachyFramgia) [inline app/views/comments/_comment.html.erb]: indent

File: app/views/reviews/_form.html.erb [hunk 6]
```diff
@@ -0,0 +1,11 @@
+<%= form_for [@book, @review], html: {multipart: true} do |f| %>
+  <div class="field">
+    <div class = "star-rating-review"></div>
+    <%= f.text_field :content, placeholder: t("title"), class: "form-control" %>
+    <%= f.text_area :thesis_statement, placeholder: t("compose_new_body"),
+      class: "form-control" %>
+  </div>
+  <div class="col-md-2">
+  <%= f.submit t("post"), class: "btn btn-primary" %>
```
Comment (MichalPlachyFramgia) [inline app/views/reviews/_form.html.erb]: indent

File: app/controllers/comments_controller.rb [hunk 7]
```diff
@@ -0,0 +1,16 @@
+class CommentsController < ApplicationController
+  def create
+    @review = Review.find params[:review_id]
+    @comment = @review.comments.build comment_params
+    @comment.user_id = current_user.id
```
Comment (khanhkhanh) [inline app/controllers/comments_controller.rb]: comment.user = current_user

**Pull request merged**